enhancements/agent-installer/agent-based-installer.md

```diff
@@ -0,0 +1,279 @@
+---
+title: agent-based-installer
+authors:
+  - "@zaneb"
+reviewers:
+  - "@avishayt"
+  - "@celebdor"
+  - "@hardys"
+  - "@lranjbar"
+  - "@mhrivnak"
+  - "@patrickdillon"
+  - "@pawanpinjarkar"
+  - "@romfreiman"
+  - "@rwsu"
+  - "@sdodson"
+approvers:
+  - "@dhellmann"
+creation-date: 2022-03-03
+last-updated: 2022-03-03
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/AGENT-9
+---
+
+# Agent-based Installer
+
+## Summary
+
+The agent-based installer is an installation method that will use a bootable,
+ephemeral installer image running on the hosts that are to become part of the
+cluster. The user will generate the image using a command-line tool. The image
+itself will contain components (such as
+[assisted-service](https://github.com/openshift/assisted-service/#readme) and
+[assisted-agent](https://github.com/openshift/assisted-installer-agent#readme))
+of the Assisted and Multi-Cluster Engine (MCE) installation methods. The
+installer will be usable in both a fully-automated workflow, where
+configuration is provided upfront, and an interactive workflow where the user
+can interact with a GUI similar to the hosted Assisted installation service.
+
+## Motivation
+
+Every on-premises user has a first cluster ("Cluster 0"), and many of them will
+be installed in a disconnected environment. While powerful tools exist for
+creating multiple on-premises clusters (Hive, HyperShift, MCE), they all run in
+a cluster themselves, which must somehow be provisioned without these tools.
+
+Each of the current installation options has limitations that make it
+unpalatable to a subset of users. IPI is highly opinionated, and suffers from
+providing limited visibility of configuration problems in on-premises
+environments. UPI exposes users to the full complexity of OpenShift to the user
+with little guidance. The (hosted) Assisted service is not available for
+IPv6-only or disconnected environments. For some users, none of these options
```

Comment:
It does available for dual stack and will be available to ipv6 once quay will be. 
I would limit it to disconnected?